### PR TITLE
Make a concept prototype of execvp

### PR DESCRIPTION
**Pull request by gekkowrld**

### CODE REVIEW

**Comment by gekkowrld** [comment]
Apparently getenv isn't allowed, so I'll have to find a way to do it

File: execute.c
Comment (RaymondEdibo) [inline execute.c:1]: why didn you just use execve(c, args, NULL), instead of _execvp(c);

File: execute.c [hunk 1]
```diff
@@ -19,8 +19,9 @@ void execute(char *c)
 		{
 				args[0] = c;
 				args[1] = NULL;
-				execve(c, args, NULL);
-				perror("execve");
+				/* execve(c, args, NULL); */
+				_execvp(c);
```
Comment (RaymondEdibo) [inline execute.c:23]: why not just use execve and specify the path instead of using execvp(which isn't allowed)

Comment (RaymondEdibo) [reply execute.c:23]: like this execve(c, args, NULL);

File: main.h [hunk 2]
```diff
@@ -13,5 +13,6 @@
 void prompt(void);
 ssize_t read_input(char *input, size_t max_l);
 void execute(char *c);
+void _execvp(char *input);
```
Comment (RaymondEdibo) [inline main.h:16]: you didnt push any file with the function prototype; void _execvp(char *input);

**Review by RaymondEdibo — CHANGES_REQUESTED** [review]

File: execute.c (hunk 1 above)
Comment (gekkowrld) [reply execute.c:23]: I just found out the env part of execve, I'll make the appropriate changes

File: main.h (hunk 2 above)
Comment (gekkowrld) [reply main.h:16]: This was a brainstorming kinda thing so I didn't bother adding it

Comment (RaymondEdibo) [reply main.h:16]: okay

**Comment by gekkowrld** [comment]
Changes have been pushed to pr #2

**Pull request closed without merging**